pkg/sql/catalog/schemadesc/virtual_schema_desc.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2020 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package schemadesc
+
+import (
+	"github.com/cockroachdb/cockroach/pkg/sql/catalog"
+	"github.com/cockroachdb/cockroach/pkg/sql/catalog/catconstants"
+	"github.com/cockroachdb/cockroach/pkg/sql/catalog/descpb"
+)
+
+// GetVirtualSchemaByID returns a virtual schema with a given ID if it exists.
+//
+// The returned descriptor carries only a basic functionality, requiring the
+// caller to check the SchemaKind to determine how to use the descriptor. The
+// returned descriptor is not mapped to a database; every database has all of
+// the same virtual schemas and the ParentID on the returned descriptor will be
+// descpb.InvalidID.
+func GetVirtualSchemaByID(id descpb.ID) (catalog.SchemaDescriptor, bool) {
```

Comment:
Would it be possible to pass the parent ID to `GetVirtualSchemaByID` and build the descriptor object with that parent ID? I feel uneasy about having the same descriptor for a whole class of schemas, somehow.